app/decorators/proxy_rule_decorator.rb

```diff
@@ -5,7 +5,10 @@ class ProxyRuleDecorator < ApplicationDecorator
 
   def pattern
     pattern_value = object.pattern
-    backend_api_path ? File.join('/', backend_api_path, pattern_value) : pattern_value
+    return pattern_value unless backend_api_path
+    parts = ['/', backend_api_path]
+    parts << pattern_value unless pattern_value == '/'
+    File.join(*parts)
```

Comment:
Question: is it not better to manipulate the path with the rails `UrlHelper` ?

Reply:
I agree with @didierofrivia , this way it seems that it is somehow related to a File, but it isn't :thinking: 
another alternative would be `parts.join('/')`

Reply:
Spoken with Gui. I will push this fix after lunch in another PR :+1: